start_jupyter.py

```diff
@@ -64,20 +64,11 @@ def printflush(x):
 # This is only if the user is NOT starting the singularity container
 # (for singularity, starting jupyter is done in _xentenv_inner)
 START_JUPYTER = """
-# Arcane conda setup instructions
-if [ -f "{conda_dir}/etc/profile.d/conda.sh" ]; then
-    echo "Using slightly less magic conda setup"
-    . "{conda_dir}/etc/profile.d/conda.sh"
-else
-    echo "Using totally unmagical conda setup"
-    export PATH="{conda_dir}/bin:$PATH"
-fi
-
-source {conda_dir}/bin/activate {env_name}
-
 JUP_PORT=$(( 15000 + (RANDOM %= 5000) ))
 JUP_HOST=$(hostname -i)
-{conda_dir}/envs/{env_name}/bin/jupyter {jupyter} --no-browser --port=$JUP_PORT --ip=$JUP_HOST 2>&1
+{bleeding_edge_sh}
```

Comment:
Sorry, maybe it wasn't clear. This is a totally new bleeding edge that uses the deployHQ-managed directories on `/dali/lgrandi/xenonnt/software`, which are synced with the master branches of the packages from GitHub. This is how we historically referred to things as bleeding edge.

Reply:
> 
> 
> @JoranAngevaare can you maybe comment more (or DM me on slack) on why installing over base_env doesn't work? It's my first time hearing about this and sounds like maybe we should improve the instructions or something.


Sure, here is a [gist](https://gist.github.com/JoranAngevaare/9f0d6376c80e3cb8253e48f167bc9a80). 


> In principle you are right, it probably doesn't hurt. I'm just a bit hesitant because I think it's important that developers work on top of our main environments especially for anything related to analysis (which anything using env_starter likely does).


I've always done things like this (and did file quite some pull requests 😉) but I don't recall this ever being an issue